besu/src/test/java/org/hyperledger/besu/services/BesuEventsImplTest.java

```diff
@@ -436,7 +432,7 @@ public void transactionDroppedEventDoesNotFireAfterUnsubscribe() {
     transactionPool.addLocalTransaction(TX2);
 
     assertThat(result.get()).isNotNull();
-    serviceImpl.removeTransactionAddedListener(id);
+    serviceImpl.removeTransactionDroppedListener(id);
```

Comment:
Not sure how this test ever passed since it was removing a different listener than it was adding.